database/query/entitlements.sql

```diff
@@ -4,4 +4,9 @@
 -- name: GetFeatureInProject :one
 SELECT f.settings FROM entitlements e
 INNER JOIN features f ON f.name = e.feature
-WHERE e.project_id = sqlc.arg(project_id)::UUID AND e.feature = sqlc.arg(feature)::TEXT;
\ No newline at end of file
+WHERE e.project_id = sqlc.arg(project_id)::UUID AND e.feature = sqlc.arg(feature)::TEXT;
+
+-- name: GetEntitlementsByProjectID :many
+SELECT *
+FROM entitlements
+WHERE project_id = sqlc.arg(project_id)::UUID;
```

Comment:
I'm guessing if this was `SELECT feature FROM entitlements` you still wouldn't get a `[]string` as the return type...

Reply:
It actually does. I did it this way to be more generic and reusable in the future but if you think it's better for this one to only return a slice of the feature names then I'm ok with it.

Reply:
Done, thanks